### ISSUE TITLE
Set up DNS for heart.museuminabox.org

### ISSUE THREAD

**Issue by amcewen** [museuminabox/museuminabox#49]
Now we've got a website that sits on www.museuminabox.org, we need a new hostname to point to this cloud service.

I think it makes sense to have the Brains talk to api.museuminabox.org, but I'm wondering if we should have another synonym for it as a better URL for pointing people at.  I.e. "the website is www.museuminabox.org, but if you want to see recent boops they're at cloud.museuminabox.org/boops".

Obviously it doesn't have to be cloud.museuminabox.org, but pointing people at api.museuminabox.org doesn't feel right.  

warehouse.museuminabox.org because that's where things are stored?  :-)  Or depository.museuminabox.org... 

Anyway, once we've decided what hostnames we want, we need to ask Laughing Squid to set them up as CNAME records to point to museuminabox.herokuapp.com on the DNS for us.

Then we need to configure heroku to respond to those names (I've added api.museuminabox.org already for that) and update the metadata that gets written into the NFC tags (only for when you hold them against an Android/non-Apple NFC phone, as those URLs will be wrong at present) and update the Brains to the right hostname (I'll raise a separate issue over in the Brain repo momentarily)

**Comment by amcewen** [comment]
It'd be useful to get this updated, and maybe also choose another synonym too, as then we can enable HTTPS on the Heroku app (now that we've moved off the free dyno)...

**Comment by amcewen** [comment]
The Django changes (if there are any) can be tracked in https://github.com/museuminabox/museuminabox-v2/issues/24 so once the DNS update is done, this issue can be closed.

**Comment by george08** [comment]
OK, so I think we've decided on heart.museuminabox.org ?

So I'll ask Laughing Squid to this subdomain. Anything else we need from them?

**Comment by george08** [comment]
Ticket initiated.

**Comment by amcewen** [comment]
I don't think there's anything else we need.  All sounds good to me.

**Comment by george08** [comment]
Scott Devaney, Feb 28, 10:13 EST:

> Hello George,
> 
> I have added the following DNS records for you as requested:
> 
> heart.museuminabox.org CNAME heart.museuminabox.org.herokudns.com
> 
> Please allow up to 24 hours for these changes to propagate to all locations.
> 
> Thanks!
> Scott

**Comment by george08** [comment]
@amcewen do feel free to assign issues to people ;)
